ncm-ccm/src/main/pan/components/ccm/schema.pan

```diff
@@ -35,7 +30,7 @@ type kerberos_principal_string = string with {
         error("Not a valid principal " + principal);
     };
 
-    realm = pc_r[len-1];
+    realm = pc_r[len - 1];
```

Comment:
it looks a bit odd, but i'd rather have panlint complain about this then no panlint. feel free to open an issue 😉